README.md

```diff
@@ -1,8 +1,48 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Omair Farooqui . Jarvis Consulting
 
+I graduated from Ontario Tech University with a BEng in Software Engineering. During my undergraduate studies, I was able to learn many key engineering concepts, some of which include Object Oriented Programming and Design, Data Structures, Design and Analysis of Algorithms, Software Design and Architecture, and Data Management Systems. During my undergraduate studies, I worked on numerous different projects while using many different technologies (Java, Python, C, Android, SQL, JS/HTML/CSS etc.) and collaborated with many different teams. This allowed me to learn how to adapt to different working environments with ease and quickness. In these school projects, I was given the opportunity to display my leadership skills as well, as I was in charge of conducting weekly meetings along with making sure my team had met deadlines and stayed on track with the project timeline. The responsibility of being the group leader also resulted in lots of important decision making and keeping constant communication with team members to ensure that everyone was completing assigned tasks and on the same page. For the last few months, I have been working with Jarvis as a Software Developer/Data Engineer and have been able to work with many engineering technologies (Docker, Git, Google Cloud Platform, Linux, Bash, PostgreSQL, Core Java etc.) and practice developing in an agile software development lifecycle. I am looking forward to using my knowledge and skills to provide creative solutions on projects that will reflect a positive impact on real world problems. I am excited to grow as a Software Engineer and absorb all the knowledge and experience that comes my way.
```

Comment:
I graduated from Ontario Tech University with a BEng in Software Engineering. During my undergraduate studies, I was able to learn many key engineering concepts, some of which include Object Oriented Programming and Design, Data Structures, Design and Analysis of Algorithms, Software Design and Architecture, and Data Management Systems. During my undergraduate studies, I worked on numerous different projects while using many different technologies (Java, Python, C, Android, SQL, JS/HTML/CSS etc.) and collaborated with many different teams. This allowed me to learn how to adapt to different working environments with ease and quickness. In these school projects, I was given the opportunity to display my leadership skills, as I was in charge of conducting weekly meetings and making sure my team had met deadlines and stayed on track with the project timeline. Being the group leader also resulted in lots of important decision-making and constant communication with team members to ensure that everyone was completing assigned tasks and on the same page. For the last few months, I have been working with Jarvis as a Software Developer/Data Engineer and have been able to work with many engineering technologies (Docker, Git, Google Cloud Platform, Linux, Bash, PostgreSQL, Core Java etc.) and practice developing in an agile software development lifecycle. I am looking forward to using my knowledge and skills to provide creative solutions on projects that will positively impact real-world problems. I am excited to grow as a Software Engineer and absorb all the knowledge and experience that comes my way.